.woodpecker/.test-pr.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  install:
+    image: danlynn/ember-cli:4.8.0
+    commands:
+      - npm ci
+  lint:
+    image: danlynn/ember-cli:4.8.0
+    group: lint
+    commands:
+      - npm run lint
+  test:
+    image: danlynn/ember-cli:4.8.0
+    commands:
+      - npm run test:ember
+    failure: ignore
```

Comment:
Was there a specific reason why this was added? Or copy-pasta from a different setup?

```suggestion
```

Reply:
This was indeed a copy-paste problem